.github/workflows/test-reporter.yml

```diff
@@ -0,0 +1,20 @@
+---
+## Workflow to process the JUnit test results and add a report to the checks.
+name: test-reporter
+on:
+  workflow_run:
+    workflows:
+      - test-linux
+    types:
+      - completed
+
+jobs:
+  report:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: elastic/apm-pipeline-library/.github/actions/test-report@current
+        with:
+          artifact: test-results            # artifact name
```

Comment:
Linking for visibility: https://github.com/elastic/apm-pipeline-library/issues/2063 these can quite noisy but hit an issue with reporting only failed.

Reply:
https://github.com/elastic/apm-agent-dotnet/pull/2019 is the one testing that particular test reporter action

Reply:
although it won't test it since the action needs exist in the default main branch. I'll try with the ecs-logging-net repo